src/app/map-view/map-view.component.css

```diff
@@ -1,21 +1,79 @@
-*{
-  background-color: #7cc254;
-  margin: 0 0;
+.row {
+  margin-top: 1vh;
+  display: flex;
+}
+
+app-tracks {
+  flex: 30%;
+  margin-right: 0.5vw;
 }
 
-div.map{
-  float: right;
-  width: 80%;
-  margin-right: 5vw;
+.viewport{
+  background-color: #7cc254;
+  height: 100%;
+  flex: 70%;
+  margin: 0;
+  text-align: center;
+  font-size: 1.5vw;
 }
 
-#info{
-  padding: 0 0 40% 0.5%;
-  font-size: 0.9vw;
+.viewport.p {
+  padding: 0;
 }
 
-#pilot{
-  font-size: 1.1vw;
+.track-header{
+  font-size: 1.2vw;
   text-align: center;
-  padding: 1vh 0;
+  padding: 1vh 1vw;
+}
+
+.track-info-container {
+  display: flex;
+}
+
+.track-info {
+  flex: 33%;
+  font-size: 1vw;
+  text-align: left;
+}
+
+.tooltip-container {
+  visibility: hidden;
+  position: relative;
+  background-color: transparent;
+  width: 1px;
+  height: 5px;
+  z-index: 100;
+}
+
+.tooltip-container .tooltip-text {
+  visibility: inherit;
+  width: 11.5vw;
+  background-color: rgba(0,60,136,.5);
+  color: #fff;
+  text-align: left;
+  font-size: 1vw;
+  border-radius: 5px;
+  padding: 3px;
+  position: absolute;
+  z-index: 101;
+  bottom: 175%;
+  left: 50%;
+  margin-left: -6vw;
+  pointer-events: none;
+}
+
+.tooltip-container .tooltip-text::after {
+  content: " ";
+  position: absolute;
```

Comment:
Was the absolute position necessary? It's better to try not to use it when possible

Reply:
Yes it is, it's absolute, but relative to it's parent so that's completely fine